### PR TITLE
Updates To The Read Me

### PR DESCRIPTION
**Pull request by marlak**
Added Markdown in addition to more information.

### CODE REVIEW

File: web-js/stream-service-uniqueValueRenderer/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+In order to see the service, the Simulator must be running.
+
+##Usage notes:
+This sample demonstrates how to apply a unique value renderer on a Stream Service.
+```javascript
+var uniqueValueRenderer = new UniqueValueRenderer(new SimpleMarkerSymbol("circle", 8,
+            new SimpleLineSymbol("solid",
+            new Color( [255, 0, 0, 0] ), 1),
+            new Color( [255, 0, 0, 0.4] )
+          ), "CatergoryCode");
```
Comment (jgravois) [inline web-js/stream-service-uniqueValueRenderer/README.md]: "CategoryCode"

**Comment by jgravois** [comment]
looks good @marlak!

**Comment by marlak** [comment]
@jgravois updated let me know what you think :smiley:

**Comment by jgravois** [comment]
looks good, except for the extra 'r' in Category :smile:

**Comment by marlak** [comment]
@jgravois All squashed/reverted and what not.  :octocat: Credit goes to @nohe427! :smiley:

**Comment by jgravois** [comment]
:+1:

**Comment by jgravois** [comment]
it looks like you probably created a new branch and pushed directly upstream somewhere in the process of creating this PR.

can you confirm that Esri/[readmeupdates](https://github.com/esri/developer-support/tree/readmeupdates) is definitely a relic with nothing worth saving and delete it (from [here](https://github.com/Esri/developer-support/branches)) if appropriate?

**Pull request merged**